hello-core-logic/createListMimicTrello.js

```diff
@@ -0,0 +1,41 @@
+//  Create List
+// Write a function named createList that takes two arguments, boardName and listName. The function should create a list with name
+// listName within board boardName with no cards to start. If the boardName doesn't exist, it should return Board doesn't exist
+// and, if a list with the same name exists already, it should return List name already exists.
+
+const hello = {
+    'Tester Board': {
+      'To Do': ['Laundry', 'Buy Apples', 'Pay Phone Bill'],
+      'Doing': ['Laundry', 'Studying Javascript', 'Studying HTML', 'Studying Ruby'],
+      'Done': ['Laundry']
+    },
+    'Dreams': {
+      'Wish List': ['Conquer the Seven Kingdoms', 'Get my baby back', 'My hand needs to chill'],
+    },
+    'Cuisine': {
+      'Region':['African', 'European', 'Brazilian', 'Asian'] }
+};
+
+board = "Cuisine";
+list = "New List";
+
+function createList(boardName, listName) {
+  let arrKeys = Object.keys(hello);
+    if (arrKeys.includes(boardName)) {
+      let arrSubKeys = Object.keys(hello[boardName]);
+      if (!arrSubKeys.includes(listName)) {
+        hello[boardName][listName] = [];
+        message = `The list "${listName}" was created!`;
+      } else {
+        message = `The name of List "${listName}" already exists. It's not possible to add it!`;
```

Comment:
A return statement not in a `forEach` loop will return from the code immediately and stop code execution. 
There is no need for you to store your string in `message` to then return it on the next line.

You can do all of that on one line

```javascript
return `The name of List "${listName}" already exists. It's not possible to add it!`;
```